docs/docs/recipes/deployment/README.md

```diff
@@ -16,11 +16,11 @@ You can set env directly or put a `.env` file inside the Logto project root. If
 - `PORT` The port that Logto listens to. Default `3001`.
 - `ENDPOINT` You may specify a URL with your custom domain for production (E.g. `ENDPOINT=https://logto.domain.com`). This will affect the value of the [OIDC issuer identifier](https://openid.net/specs/openid-connect-core-1_0.html#IssuerIdentifier) and Admin Console Redirect URIs.
 - `OIDC_COOKIE_KEYS` The string array of the [signing cookie keys](https://github.com/panva/node-oidc-provider/blob/main/docs/README.md#cookieskeys). Rotate regularly to keep safety.
-- `OIDC_PRIVATE_KEY` The content of the private key for [OIDC JWT signing](https://openid.net/specs/openid-connect-core-1_0.html#Signing). If you'd like to set this in `.env`, you can leverage [multiline values](https://github.com/motdotla/dotenv#multiline-values) support.
+- `OIDC_PRIVATE_KEYS` The string array of the content of the private key for [OIDC JWT signing](https://openid.net/specs/openid-connect-core-1_0.html#Signing). If you'd like to set this in `.env`, you can leverage [multiline values](https://github.com/motdotla/dotenv#multiline-values) support.
```

Comment:
If we use array here, we'd better to put an example otherwise it's hard to understand the exact format.